server/routes/isUploaded.js

```diff
@@ -7,15 +7,16 @@ app.use(cors());
 app.use(bodyParser.json());
 // import { minioClient } from '../minioConfig.js';
 import { file_stats } from '../Database_queries/queries.js';
+import { logger, log } from '../logger.js';
 
 router.get('/:url', async function (req, res) {
   try {
     const filename = req.query.fileName;
     const response = await file_stats('datadrive-dev', filename.split('.')[0]);
     res.json({ isUploaded: response[0]?.isUploaded });
   } catch (error) {
-    console.error('Error:', error);
+    log.error('Error:', error);
```

Comment:
The error logging should use `log.error` instead of `console.error` to maintain consistency with the new logging mechanism introduced by Winston. This ensures that all error logs are handled by the logger. 
```diff
-    console.error('Error:', error);
+     log.error('Error:', error);
```
*This comment was auto-generated by **Code Bot** to provide suggestions and improvements.*